code/src/java/pcgen/rules/persistence/CDOMSubLineLoader.java

```diff
@@ -111,36 +109,13 @@ public T getCDOMObject()
 		throw new IllegalArgumentException();
 	}
 
-	public String getPrefix()
+	String getPrefix()
 	{
 		return targetPrefix;
 	}
 
-	public Class<T> getLoadedClass()
+	Class<T> getLoadedClass()
 	{
 		return targetClass;
 	}
-
-	public void unloadObject(LoadContext lc, T object, StringBuilder sb)
```

Comment:
Was written for future used of unloading (related to the getExtractURI method above)